.github/workflows/npm.yml

```
@@ -0,0 +1,24 @@
+        - 'v*'
+    workflow_dispatch:
+
+jobs:
+    publish:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v4
+            - uses: DeLaGuardo/setup-clojure@12.5
+            - uses: actions/setup-node@v3
+              with:
+                node-version: "20"
```

yamllint

[error] wrong indentation: expected 18 but found 16

(indentation)